fix_includes.py

```diff
@@ -2146,20 +2153,26 @@ def FixFileLines(iwyu_record, file_lines, flags):
       #    'namespace foo { class Bar; }\n' -> 'namespace foo {\nclass Bar;\n}'
       # along with collecting multiple classes in the same namespace.
       new_lines = _NormalizeNamespaceForwardDeclareLines(new_lines)
+
+    # Add line seperators to the new_lines.
```

Comment:
`new_lines` -> `new lines`. I think you can leave out the `Strip the ends...` part of this comment, that detail is obvious from the code.

I would prefer this whole thing as a list comprehension:

    new_lines = [nl.rstrip() + fileinfo.linesep for nl in new_lines]

Reply:
Fixed in latest